api/src/main/java/io/opentelemetry/metrics/Meter.java

```diff
@@ -202,4 +47,14 @@
    * @since 0.1.0
    */
   BatchRecorder newBatchRecorder(String... keyValuePairs);
```

Comment:
why wouldn't this go onto the RecorderInstruments interface? Likewise with the newBatchObserver and the ObserverInstruments interface?

Reply:
From the example in the spec ([link](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#batch-observer)), I thought that `newBatchRecorder` and `newBatchObserver` can only be created by the `Meter`

Reply:
The Meter interface, as you've defined it, extends both of those interfaces, though...so it's on the Meter either way.

Reply:
`BatchObserver` extends `ObserverInstruments`. If we move `newBatchObserver` inside `ObserverInstruments`, a `BatchObserver` can generate new `BatchObserver`s. Is this what the metric sdk should provide?

Reply:
Ugh. I hadn't read that spec carefully enough to realize that you need to be able to create instruments with the batch observer, and not just associate them. I wonder if this spec has been implemented anywhere else but golang yet.

Reply:
My opinion is that, since those methods just delegate to the MeterSdk anyway, that we should get rid of that from the spec, and have it be an optional thing for APIs to expose, rather than mandatory. I'll open up a spec issue for it.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/pull/1030